tasks/quality_control/task_fastp.wdl

```diff
@@ -0,0 +1,89 @@
+version 1.0
+
+task fastp {
+  input {
+    File read1
+    File read2
```

Comment:
Looks great as is, but we may want to make an option for single-end read data in the future

Reply:
Disregard. Didn't see the se option below!